### ISSUE TITLE
Number of records in list displayed

### ISSUE THREAD

**Issue by fionaborthwick1** [PTFS-Europe/rebus-list-2#51]
At UHI this morning they asked if the view of reading lists told the user how many items were on the reading list.  Presently there is just a scroll bar but it would be good if we could display the number of items on the list and even the number of items under each category.

Would this be possible?

**Comment by AndrewIsh** [comment]
Yes, good idea. I've created an issue: https://github.com/PTFS-Europe/rebus-list/issues/1413
